<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordingBusinessActionWidget</class>
 <widget class="QWidget" name="RecordingBusinessActionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Quality:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="qualityComboBox"/>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QSpinBox" name="fpsSpinBox">
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="value">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="fpsLabel">
     <property name="text">
      <string>FPS</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="beforeLabel">
     <property name="text">
      <string>Pre-Recording:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QSpinBox" name="beforeSpinBox">
     <property name="suffix">
      <string>s</string>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="afterLabel">
     <property name="text">
      <string>Post-Recording:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QSpinBox" name="afterSpinBox">
     <property name="suffix">
      <string>s</string>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QSpinBox" name="durationSpinBox">
     <property name="suffix">
      <string>s</string>
     </property>
     <property name="maximum">
      <number>86400</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <widget class="QLabel" name="durationLabel">
     <property name="text">
      <string>max</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QCheckBox" name="fixedDurationCheckBox">
     <property name="text">
      <string>Fixed duration:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="nx::client::desktop::TimeDurationWidget" name="fixedDuration">
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::client::desktop::TimeDurationWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/business/time_duration_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
